### PR TITLE
Kmp pro dino wi fi

### PR DESCRIPTION
**Pull request by kmpelectronics**
Hello,

Please, merge our board information to master branch.

Regards,
Plamen Kovandjiev
KMP Electronics Ltd

### CODE REVIEW

**Comment by kmpelectronics** [comment]
Please, merge we branch.

**Comment by igrr** [comment]
This PR seems to miss your additions to boards.txt?
Also please add the board name to package/package_esp8266com_index.template.json.

File: doc/filesystem.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ ESPresso Lite 1.0 | 4M | 1M, 3M
 ESPresso Lite 2.0 | 4M | 1M, 3M
 NodeMCU 0.9    | 4M | 1M, 3M
 NodeMCU 1.0    | 4M | 1M, 3M
-Olimex MOD-WIFI-ESP8266(-DEV)| 2M | 1M
 ProDino WiFi ESP 1.0 | 4M | 1M, 3M
```
Comment (igrr) [inline doc/filesystem.md]: Please keep this in alphabetical order

**Comment by kmpelectronics** [comment]
Hi,
Thank you for your help.
I create now pull request with fixed issues.

Regards,
Plamen Kovandjiev
KMP Electronics Ltd

**Pull request closed without merging**